mock/api.js

```diff
@@ -6,41 +6,24 @@ const DEFAULT_SIZE = 100;
 export const mockIndices = new Array(DEFAULT_SIZE).fill().map(() => moment().format('YYYY-MM'));
 
 export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-  controller: casual.word + index,
-  key: casual.word + index,
-  last_modified_string: moment.utc() + index,
+  controller: `${casual.word}.${casual.word}.com`,
+  key: casual.ip,
```

Comment:
I don't think we use `key` for anything, but React complains if arrays don't have a "unique key"; so this is probably just fine. However the real code (the current server code as well as the original dashboard model) will set this to the controller name.

Reply:
I'd prefer that the mocks try to model the form and semantics of the real APIs; it may not matter now, where the dashboard code doesn't care, but it might matter in the future and it's likely easier to get it right now than to make someone figure out what's going on in a year when the processing changes. :-)

Unfortunately, that means converting this into a full lambda method with local variables so that a value can be used more than once.